data/styles/clear/include/Roads.mapcss

```diff
@@ -1173,33 +1197,50 @@ line|z14-[railway=yard][tunnel?]
 
 /* 9.3 Rail bridge 14-22 ZOOM */
 
+/* TODO
+  remove following rules
+  add casing-color and z-index definitions
+  to test these new bridge styles
+*/
+line|z14-[railway=tram][bridge?]::bridgewhite,
+line|z14-[railway=monorail][bridge?]::bridgewhite,
+line|z14-[railway=funicular][bridge?]::bridgewhite,
+line|z14-[railway=narrow_gauge][bridge?]::bridgewhite,
+line|z14-[railway=tram][bridge?]::bridgeblack,
+line|z14-[railway=monorail][bridge?]::bridgeblack,
+line|z14-[railway=funicular][bridge?]::bridgeblack,
+line|z14-[railway=narrow_gauge][bridge?]::bridgeblack,
+{casing-linecap: round;}
```

Comment:
So, we we put this block (somewhere before) with current eval logic, everything also will be fine?

Reply:
It will not change anything then.
But if we copy-paste the evals section into a new file, then we'll need this block to stop "new" untested bridge styles showing up.